### PR TITLE
Add manual testing section to DG

### PR DESCRIPTION
**Pull request by jemehgoh**
This PR adds the manual testing section (and updates to the NFRs) for the DG.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,13 +66,17 @@ Adds an event to the event list, a participant to an event, or an item to an eve
...
 Format:  
 
-* `add -e EVENT -t TIME -v VENUE` for adding an event to the events list.
+* `add -e EVENT -t TIME -v VENUE -u PRIORITY` for adding an event to the events list.
 * `add -p PARTICIPANT -n NUMBER -email EMAIL -e EVENT` for adding a participant to an event.
 * `add -m ITEM -e EVENT` for adding an item to an event.
 
+Remarks:
+* `TIME` must be entered in the format `yyyy-mm-dd HH:mm`.
+* `PRIORITY` must be either `HIGH`, `MEDIUM`, or `LOW`.
```
Comment (LTK-1606) [inline docs/UserGuide.md:75]: user input for priority is not case-sensitive

**Review by LTK-1606 — COMMENTED** [review]
Overall LGTM, just a few things

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -636,8 +636,11 @@ The user is able to organise and manage his events more quickly and efficiently
 
 ## Non-Functional Requirements
 
-* Should work for any **mainstream OS** as long as Java 17 is installed.
-* Should be able to store data for up to 1000 events without any loss in performance.
+* Should work for any **mainstream OS** (Windows, MacOS, Linux) as long as Java 17 is installed.
+* Should be able to store data for up to 1000 `Event`s without any loss in performance.
```
[the remark this answers is not in the document]
Comment (jemehgoh) [reply docs/DeveloperGuide.md:640]: I took this from the AB3 developer guide, will change this to "a large number".

File: docs/UserGuide.md (hunk 1 above)
Comment (jemehgoh) [reply docs/UserGuide.md:75]: Noted, will amend accordingly.

**Pull request merged**